### ISSUE TITLE
Bug with -init-mask

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#1323]
~~~
sct_propseg -i TIA03_MPR_BF.nii.gz -c t1 -init-mask viewer

--
Spinal Cord Toolbox (master/93295e61a30518cc824d79c087b15e7af47cbcbd)
Running /Users/julien/code/sct/scripts/sct_propseg.py -i TIA03_MPR_BF.nii.gz -c t1 -init-mask viewer
Traceback (most recent call last):
  File "/Users/julien/code/sct/scripts/sct_propseg.py", line 314, in <module>
    arguments = parser.parse(args)
  File "/Users/julien/code/sct/scripts/msct_parser.py", line 363, in parse
    dictionary[arg] = self.options[arg].check_integrity(param)
  File "/Users/julien/code/sct/scripts/msct_parser.py", line 134, in check_integrity
    return self.checkIfNifti(param)
  File "/Users/julien/code/sct/scripts/msct_parser.py", line 221, in checkIfNifti
    elif param.lower() in self.list_no_image:
TypeError: argument of type 'NoneType' is not iterable
~~~

**Comment by peristeri** [comment]
This bug will be fixed in the new version of the viewer.

**Comment by SaraDupont** [comment]
The issue persists on branch `viewerQt`.

**Comment by SaraDupont** [comment]
I provided a fix on branch `sd_issue_1323` (see PR #1357) 

With that fix, the option `-init-mask` of `sct_propseg` works but the position of the slices displayed by the viewer is not adapted to the use of `-init-mask` in `sct_propseg` (the viewer displays only three slices from the top of SC).
